lint.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
+set -ex
+
 find . -type f -name "*.sh" -print0 | xargs -0 shellcheck
-find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which isort)" --check-only --line-length 140
-find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which pycodestyle)" --config=.pycodestyle
-find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(pyenv which pylint)" --load-plugins pylint_quotes
+find . -type f -name "*.py" -print0 | PYTHONPATH=. xargs -0 "$(command -v isort)" --check-only --line-length 140
```

Comment:
`command`? 😅  what's that?

Reply:
`command -v` is apparently more widely distributed `which` (shellcheck complained about which)